locksmith/src/controllers/v2/checkoutController.ts

```diff
@@ -113,9 +120,8 @@ export const deleteCheckoutConfig: RequestHandler = async (
   const deleted = await deleteCheckoutConfigOperation(userAddress, id)
 
   if (!deleted) {
-    return response.status(403).send({
-      message:
-        'Unauthorized: You do not have permission to delete this configuration.',
+    return response.status(404).send({
```

Comment:
Let's keep 403 here. 404 would be for "not found" . If needed we can add a check to see if the config exists?